Marsiple/App/New Group/PhotoCollectionViewController.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  PhotoCollectionViewController.swift
+//  Marsiple
+//
+//  Created by Matej Korman on 20/07/2018.
+//  Copyright © 2018 Matej Korman. All rights reserved.
+//
+
+import UIKit
+
+class PhotoCollectionViewController: UICollectionViewController {
+    private let photos: [UIImage]
+    
+    init(photos: [UIImage]) {
+        self.photos = photos
+        super.init(collectionViewLayout: UICollectionViewFlowLayout.init())
+        setupCollectionView()
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    // MARK: UICollectionViewDataSource
+    override func numberOfSections(in collectionView: UICollectionView) -> Int {
+        return 1
+    }
+    
+    override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return photos.count
+    }
+
+    override func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
+        guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "PhotoCollectionViewCell", for: indexPath) as? PhotoCollectionViewCell else { return UICollectionViewCell() }
+        cell.updateCell(withImage: photos[indexPath.item])
+        return cell
+    }
+}
+
+private extension PhotoCollectionViewController {
+    func setupCollectionView() {
+        self.collectionView!.backgroundColor = .white
+        self.collectionView!.register(PhotoCollectionViewCell.self, forCellWithReuseIdentifier: "PhotoCollectionViewCell")
+        if let flowLayout = self.collectionView!.collectionViewLayout as? UICollectionViewFlowLayout {
```

Comment:
We should `never/ever` (with some exceptions, which this is not) force-unwrap anything. Use optional-chaining here.